packages/template-retail-react-app/app/partials/product-view/index.jsx

```diff
@@ -393,9 +405,15 @@ const ProductView = ({
                                 </Text>
                             </Fade>
                         )}
-                        <Box display={['none', 'none', 'none', 'block']}>
-                            {renderActionButtons()}
-                        </Box>
+                        {!isProductASet && (
+                            <Box
+                                display={
+                                    isProductPartOfSet ? 'block' : ['none', 'none', 'none', 'block']
+                                }
+                            >
```

Comment:
What do you think of `isProductASet` and `isProductPartOfSet`? How are they compared to the existing terminology seen in Business Manager and documentation (they use "product set" and "set product" to mean parent and child product, respectively)?

Reply:
I assume that the reason they chose to `set product` is because it's short and its reversal of `product set` implicitly gives it a connection between the two types. 

That being said, if you read "set product" in isolation, likelihood that you would understand what it means is low. So although I normally like what old ocapi and their doc's, I think this isn't one of them. 

I'm not too concern either way, so you can do what you think is best on this topic, because we can change it since it's in the template. 

Another option (because I like to muddy the waters) would be something like `product.isSet` and `product.belongsToSet`